packages/now-cli/src/commands/env/add.ts

```diff
@@ -63,6 +69,33 @@ export default async function add(
     envTargets.push(envTarget);
   }
 
+  if (envType && !isValidEnvType(envType)) {
+    output.error(
+      `The Environment Variable type ${param(
+        envType
+      )} is invalid. It must be one of: ${getEnvTypePlaceholder()}.`
+    );
+    return 1;
+  }
+
+  if (!envType) {
+    const { inputEnvType } = (await inquirer.prompt({
+      name: 'inputEnvType',
+      type: 'list',
+      message: `Which type of Environment Variable do you want to add?`,
+      choices: [
+        { name: 'Plaintext', value: ProjectEnvType.Plaintext },
+        {
+          name: `Secret (can be created using ${code('vercel secret add')})`,
```

Comment:
```suggestion
          name: `Secret (can be created using ${getCommandName('secret add')})`,
```

Reply:
Updated in https://github.com/vercel/vercel/pull/5372/commits/0d191114447a377d4c3782c24a9e51ef095256a5.